simonfabcic/32_-_NestJS_-_ReactJS-app
# Tests (get tokens, get_user)

[What?]
Tests:
- Does view return access and refresh token
- Can logged-in users get profiles
- Is access to un-authorized users forbidden to get a profile

[Why?]
- Checking login functionality
- Difference access between logged and un-logged user

[How?]
- Check login:
    - send the login request and check if in response are "access" and "refresh" tokens
- Profiles access:
    - create user
    - create a token for the user
    - create extra client `client_with_access_token`, to which I set the `HTTP_AUTHORIZATION` header
    - try to get a profile by this client - success
    - try to get a profile with default client (unauthorized) - failure
